stdlib/opal-platform.rb

```diff
@@ -3,13 +3,10 @@
 nashorn         = `typeof(Java) !== "undefined" && Java.type`
 headless_chrome = `typeof(navigator) !== "undefined" && /\bHeadlessChrome\//.test(navigator.userAgent)`
 
-case
-when nashorn
-  OPAL_PLATFORM = 'nashorn'
-when node
-  OPAL_PLATFORM = 'nodejs'
-when headless_chrome
-  OPAL_PLATFORM = 'headless-chrome'
-else # possibly browser, which is the primary target
-  OPAL_PLATFORM = nil
-end
+OPAL_PLATFORM = if nashorn
+                  'nashorn'
+                elsif node
+                  'nodejs'
+                elsif headless_chrome
+                  'headless-chrome'
+                end
```

Comment:
I find the original form much better, also the comment disappeared

Since I also don't like very much all the other changes of this kind (mostly `case/when`) can we ditch this rule altogether?